sleap_io/model/skeleton.py

```
@@ -329,3 +329,58 @@ def add_symmetry(
+                names are not found in the skeleton.
+        """
+        if type(name_map) == list:
+            if len(name_map) != len(self.nodes):
+                raise ValueError(
+                    "List of new node names must be the same length as the current "
+                    "nodes."
+                )
+            name_map = {node: name for node, name in zip(self.nodes, name_map)}
+
+        for old_name, new_name in name_map.items():
+            if type(old_name) == Node:
```

Ruff

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)